api-migration.md

```diff
@@ -15,6 +15,29 @@ The library continues to support legacy plugins. They can be used in conjunction
 with new plugins. Your plugins list may contain any number of new plugins but
 only ever one legacy plugin.
 
+It's important to note that the `cloudant.request` object is a pass through
+stream and isn't always suitable for piping. However, the response object
```

Comment:
`isn't always suitable` - should we explain when it is suitable? Or is it better to just say something like `isn't a reliable solution for piping`.

Reply:
yeah - I guess I want to clarify that it can't be used in an event hook.